handbook/product/design/index.md

```diff
@@ -67,6 +67,34 @@ While design does adds time at the beginning of a project, it improves velocity
 
 Our design process is documented here: **[Design process](design_process.md)**
 
+## Design Team Operational Guidelines
```

Comment:
```suggestion
## Design Principles
```

I'd prefer to rename this to "Design principles" – "Operational guidelines" seems more like "the tools, processes we use" than "how we make decisions."

Reply:
I agree operational guidelines would describe the process surrounding design at Sourcegraph. This is focused on how we make decisions about design.